code/modules/food/food/snacks.dm

```diff
@@ -16,6 +16,7 @@
 	var/nutriment_amt = 0
 	var/list/nutriment_desc = list("food" = 1)
 	var/datum/reagent/nutriment/coating/coating = null
+	var/sealed = 0
```

Comment:
this should be FALSE; use defines to denote booleans. rest of the file should be TRUE instead of 1

Reply:
Done!